test/client/test_utils.py

```diff
@@ -0,0 +1,14 @@
+from unittest.mock import patch
+
+import nlpsandbox
+from nlpsandboxclient import utils
+
+
+def test_get_api_configuration():
+    """Test getting API configuration"""
+    configuration = nlpsandbox.Configuration()
+    with patch.object(nlpsandbox, "Configuration",
+                      return_value=configuration),\
+         patch.object(utils, "check_url"):
+        config = utils.get_api_configuration(host="0.0.0.0")
```

Comment:
Check indentation of lines in this part of the file

Reply:
It's on purpose to increase readability of code.  Actually context managers are a huge pain until python 3.9 where they support parenthesis around context managers.